### PR TITLE
Quickstart for Python notebook

### PR DESCRIPTION
**Pull request by brian-rose**
Replaces the old placeholder "Hello world" notebook with a quickstart teaser on basic Python usage.

### CODE REVIEW

**Comment by brian-rose** [comment]
This wraps in all the work in #50 and #48

**Comment by brian-rose** [comment]
@jukent we can pull in any further changes made in #50 into this branch. This adds a new Quickstart notebook meant to be opened and run in a Binder.

Maybe it's too long / too detailed? My goal was just to expose a curious new reader to some of the look and feel of Python code.

**Comment by jukent** [comment]
@brian-rose I think this looks great! At first I was surprised to see mentions of `numpy` and `matplotlib` but the purpose is to be a little flashy and show some things Python can't do on its own. Perhaps here we want to link to those pages once they are up.

**Comment by brian-rose** [comment]
> At first I was surprised to see mentions of `numpy` and `matplotlib` but the purpose is to be a little flashy and show some things Python can't do on its own. Perhaps here we want to link to those pages once they are up.

For sure. The idea here is just to show off some typical usage and give people a starting point if they want to play around with plotting in the Binder. I tried to avoid putting in much actual explanation of the matplotlib functions, but hope that the code to generate the line plot is pretty self-explanatory.

We can/should definitely add links to our full numpy and matplotlib tutorials once those are in place. I'll open an issue for that once we get this merged.

**Comment by brian-rose** [comment]
I haven't actually tried to run this example notebook in Binder. Probably have to wait until merging, since Binder will build a Docker image from the main branch.

**Review by jukent — APPROVED** [review]

**Comment by brian-rose** [comment]
It would be great to have at least one more set of eyes on this before merging... tagging @mgrover1

File: foundations/quickstart.ipynb [hunk 1]
```diff
@@ -0,0 +1,887 @@
+{
```
Comment (mgrover1) [inline foundations/quickstart.ipynb:1]: <p>Add quick comment mentioning start, end, # of points in between</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (mgrover1) [inline foundations/quickstart.ipynb:1]: <p>"Read on for more details..." might read better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

**Review by mgrover1 — COMMENTED** [review]
A couple of small suggestions, but this looks great!

**Review by mgrover1 — APPROVED** [review]
Thanks for making those changes! Good to merge!

**Pull request merged**